pkg/service/model_build_listener_test.go

```diff
@@ -301,3 +301,220 @@ func Test_defaultModelBuilderTask_buildListenerAttributes(t *testing.T) {
 		})
 	}
 }
+
+func Test_mergeServicePortsForListener(t *testing.T) {
```

Comment:
Are these tests out-of-date? They don't seem to match the function signature of `mergeServicePortsForListener`.

Reply:
Sorry, yes they were.  Fixed. Thanks, nice catch!